circuit-codegen/src/main/kotlin/com/slack/circuit/codegen/CircuitSymbolProcessorProvider.kt

```diff
@@ -391,11 +390,6 @@ private fun KSFunctionDeclaration.assistedParameters(
               )
             }
           }
-          type.isInstanceOf(symbols.circuitConfig) -> {
```

Comment:
This shouldn't have been available as an assisted type anyway so no real loss here